regression/acceleration/Makefile

```diff
@@ -28,5 +28,6 @@ show:
 	done;
 
 clean:
-	$(RM) tests.log
-	$(RM) */main.out
+	find -name '*.log' -execdir $(RM) '{}' \;
```

Comment:
Would `$(RM) tests.log` not do the trick here?

Reply:
I decided to do the more general and consistent approach that will remove `tests.log` as well as any others. I could change this to only the specific file if there is agreement not to use a general clean up instead?

Reply:
I agree there is now inconsistency though since some use specific commands and others general. I made the general ones when factoring out loops with limit depth. This comment is also to note that the approach should be consistent, either all use `$(RM) tests.log` or all use `find . -name '*.log' -execdir $(RM) \;`.

Reply:
Sounds good, I'm happy with either approach, as long as it is consistent. Doing the `find` in some places only suggests that those directories generate additional `.log` files, which may or may not be the case. That said, I am starting to wonder whether it wouldn't be much simpler (less code!) to move the common cleanup commands one level up. Each directory would then only have to care about any cleanup that's not covered by the parent-level `find` calls.

Reply:
Chose to go with the specific and local version. Chosen so we are clear on where log files are expected and which log files are generated by the regression tests.